tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -423,7 +437,17 @@
 			const url = getLink();
 
 			if ( url.match( /[?&]branch(es\.[^&=]*)?=/ ) ) {
-				$link.attr( 'href', url ).text( url );
+				if ( url.match( /[?&]jpcrm-populate-crm-data/ ) && ! url.match( /zero-bs-crm/ ) ) {
+					$link
+						.attr( 'href', null )
+						.text( 'Select the Jetpack CRM plugin in order to populate with CRM data' );
```

Comment:
The check was looking for `zero-bs-crm` as the prefix before the branch name rather than the checking for the plugin in the second section, however adding the query / url changes you suggested earlier makes the regex check a little better now (I think).